### PR TITLE
Create KrishJaiswal.json

### PR DESCRIPTION
**Pull request by KrishJ4856**
Add Krish's LinkFree Profile

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/2926"></a>

### CODE REVIEW

File: KrishJaiswal.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  "links": [
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/Krish4856",
+      "icon": "FaTwitter"
+    },
+    {
+      "name": "My GitHub Profile",
+      "url": "https://github.com/KrishJ4856",
+      "icon": "FaGitHub"
```
Comment (Aadarsh805) [inline KrishJaiswal.json:16]: change icon use this `FaGithub`

File: KrishJaiswal.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    },
+    {
+      "name": "I write technical blogs here",
+      "url": "https://krishwebdev.hashnode.dev",
+      "icon": "FaHashnode"
```
Comment (Aadarsh805) [inline KrishJaiswal.json:21]: change this to `SiHashnode`

**Review by Aadarsh805 — CHANGES_REQUESTED** [review]

**Comment by Aadarsh805** [comment]
@KrishJ4856 Please fix the file name. It should be {your GitHub username}.json -> KrishJ4856.json

**Review by dan-mba — CHANGES_REQUESTED** [review]
Please move your profile to the data directory

**Comment by KrishJ4856** [comment]
@Aadarsh805 @dan-mba I did the changes as told. Is it ok now?

**Review by dan-mba — CHANGES_REQUESTED** [review]
Please change your avatar to `https://github.com/KrishJ4856.png`.
This will redirect to the URL you have, but will continue working if you change your profile url

**Comment by KrishJ4856** [comment]
Done

**Review by dan-mba — APPROVED** [review]
Looks good to me

**Comment by KrishJ4856** [comment]
Thanks for your time!

**Review by Panquesito7 — APPROVED** [review]
Thanks! 🎉

**Review by eddiejaoude — APPROVED** [review]
Looks good 👍 

Great collaboration 💪 

You can also join us in the EddieHub Discord http://discord.eddiehub.org

**Pull request merged**